spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -148,6 +161,19 @@
       )
     end
 
+    class PackedRootCertificatesFinder
+      def find(_format, _id)
+        certificate_path = File.expand_path(
+          File.join(__dir__, '..', 'support', 'roots', 'yubico_u2f_root.pem')
+        )
+        [OpenSSL::X509::Certificate.new(File.read(certificate_path))]
+      end
+    end
+
+    before(:each) do
+      WebAuthn.configuration.attestation_root_certificates_finders = [PackedRootCertificatesFinder.new]
```

Comment:
What about having a spec helpers that makes this be `WebAuthn.configuration.attestation_root_certificates_finders = finder_for("yubico_u2f_root.pem")` ? :D